eZ/Publish/Core/MVC/Symfony/View/Tests/Builder/ContentViewBuilderTest.php

```diff
@@ -296,8 +381,6 @@ public function testBuildViewWithTranslatedContentWithoutLocation(): void
             ->method('getContentService')
             ->willReturn($contentServiceMock);
 
-        $this->contentViewBuilder->buildView($parameters);
```

Comment:
Removing that line seems kinda random?

Reply:
Actually, it doesn't need to be called twice, few lines below we have:
`$this->assertEquals($expectedView, $this->contentViewBuilder->buildView($parameters));`